rest/src/main/java/discord4j/rest/service/MonetizationService.java

```diff
@@ -0,0 +1,44 @@
+package discord4j.rest.service;
+
+import discord4j.discordjson.json.CreateTestEntitlementRequest;
+import discord4j.discordjson.json.EntitlementData;
+import discord4j.discordjson.json.SkuData;
+import discord4j.rest.request.Router;
+import discord4j.rest.route.Routes;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+public class MonetizationService extends RestService {
+
+    public MonetizationService(Router router) {
+        super(router);
+    }
+
+    public Flux<SkuData> getAllSkus(long applicationId) {
+        return Routes.LIST_SKUS.newRequest(applicationId)
+            .exchange(getRouter())
+            .bodyToMono(SkuData[].class)
+            .flatMapMany(Flux::fromArray);
+    }
+
+    public Flux<EntitlementData> getAllEntitlements(long applicationId) {
```

Comment:
Add support for all query params (https://discord.com/developers/docs/monetization/entitlements#list-entitlements-query-params)